lib/Rex/Commands/File.pm

```diff
@@ -97,7 +97,7 @@ use base qw(Rex::Exporter);
 
 use vars qw(%file_handles);
 
-=head2 template($file, @params)
+=head2 template($file [, @params])
```

Comment:
I originally proposed using `%params` here to be consistent with the rest of the file.

Under the hood it is used as a hash for named parameters, and similar cases elsewhere in the same file (`extract`, `sed`) were marked with `%`.

Is there a strong reason to keep this documented as an array like `@params`?

Reply:
no, will correct